tests/unit/ReferenceListTest.php

```diff
@@ -52,7 +52,7 @@ public function getConstructorArg() {
 	public function testCanConstructWithReferenceListObject() {
 		$original = new ReferenceList( array( new Reference( array( new PropertyNoValueSnak( 1 ) ) ) ) );
 		$copy = new ReferenceList( $original );
-		$this->assertSame( 1, $copy->count() );
+		$this->assertEquals( $original, $copy );
```

Comment:
I was under the impression that PHPUnit compares objects in a sane way. We are certainly using assertEquals for this in other places.